pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+description: |
+  A minimal pipeline that runs shear-specific tasks, separate from the
+  standard DRP processing.
+
+tasks:
+  makeWarp: descwl_coadd_task.MakeShearWarpTask
+  assembleCoadd:
+    # The configuration is same as the default values defined in the
+    # PipelineTasks. The purpose of overrides here is to demonstrate the
+    # configurable capacity of the pipeline and the tasks.
+    class: descwl_coadd_task.AssembleShearCoaddSlowTask
+    config:
+      connections.multiple_cell_coadd: shearCoadd
+      psf_dimensions: 21
+      remove_poisson: False
+  # TODO: Metadection goes here.
```

Comment:
```suggestion
```

I'd think not since that would be a separate task, right?

Reply:
Yes, it's a separate task. This is just a placeholder to include it in the shear pipeline when it is ready to go.